src/mappings/events/vaults.ts

```diff
@@ -71,7 +147,7 @@ export const vaultsEventKit = (block: any, data: any, module: string, path: stri
       liquidation = await saveVaultsLiquidation(payload);
     }
 
-    return [liquidation, vault.save()];
+    return [liquidation, vault.save(), dailyVaultState.save()];
   }
```

Comment:
Not sure but will it wait for `.save()` calls to complete? or should we use `await`?

Reply:
the returned array is awaited in the Calling function. so we dont have to handle it here